src/App.vue

```diff
@@ -1,15 +1,25 @@
 <template>
   <div id="app">
+    <HeaderComponent />
     <router-view></router-view>
   </div>
 </template>
 
 <script lang="ts">
+import HeaderComponent from './presentation/components/HeaderComponent.vue';
 export default {
   name: "App",
+  components: {HeaderComponent},
 }
 </script>
 
-<style scoped>
+<style>
+  * {
+    box-sizing: border-box;
+  }
 
-</style>
\ No newline at end of file
+  body {
```

Comment:
After normalise, i think it's not neccassary

Reply:
Yes, removed